templates/cicd/pipeline_cfn.yml

```diff
@@ -3,12 +3,12 @@
 AWSTemplateFormatVersion: '2010-09-09'
 Description: CodePipeline for {{$.AppName}}
 Resources:
-  {{- if eq .Source.ProviderName "Bitbucket"}}
+  {{- if or (eq .Source.ProviderName "Bitbucket") (eq .Source.ProviderName "GitHub")}}
```

Comment:
instead of doing `or`s in between types, should we add an additional template function here to simplify the logic:
https://github.com/aws/copilot-cli/blob/d0360734981d1640939412925f4c4dfd04f5996c/internal/pkg/deploy/cloudformation/stack/pipeline.go#L34

```go
var pipelineTemplateFuncs = map[string]interface{}{
	"logicalIDSafe": template.ReplaceDashesFunc,
	"isCodestarConnection": func(source interface{}) bool {
		type connectionName interface {
			ConnectionName() (string, error)
		}
		_, ok := source.(connectionName) // Does this source implement the connectionName interface?
		return ok
	}
}
```

and in the template:
```yaml
{{- if isCodestarConnection .Source }}
```